File: W4Day1/Python/IfElse1/Siblings.py
sibling1 = input()
sibling2 = input()

age1 = (int)(input())
age2 = (int)(input())

if(age1>age2):
    print(sibling1, "is older than", sibling2)
elif(age2>age1):
    print(sibling2, "is older than", sibling1)
else:
    print(sibling1, "and", sibling2, "are of the same age")

File: W4Day1/IfElse1/Siblings.java
package W4Day1.IfElse1;

import java.util.Scanner;

public class Siblings {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String sib1 = sc.nextLine();
        String sib2 = sc.nextLine();
        int age1 = sc.nextInt();
        int age2 = sc.nextInt();

        sc.close();

        if (age1 > age2) {
            System.out.printf("%s is older than %s\n", sib1, sib2);
        } else if (age2 > age1) {
            System.out.printf("%s is older than %s\n", sib2, sib1);
        } else {
            System.out.printf("%s and %s are of the same age\n", sib1, sib2);
        }
    }
}
